default/python/AI/FreeOrionAI.py

```diff
@@ -251,6 +251,7 @@ def generateOrders():  # pylint: disable=invalid-name
             print_error(e, location=action.__name__)
     main_timer.end()
     turn_timer.end()
+    turn_timer.clear()
```

Comment:
We don't need the clear and I have removed it.

The AI version of the timer clears its times when end() is called, so it does not need the clear().

The standard version of the timer does not, so that you can stop the timer and then use either of the print functions.  If it was the standard timer then it would need to be cleared once per turn in order to be a turn timer.